### PR TITLE
Access headers

### PR DESCRIPTION
**Pull request by aintgoin2goa**

### CODE REVIEW

File: server/controllers/access.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/*jshint node:true*/
+'use strict';
+
+
+
+module.exports = function(req, res, next){
+	var apiKey = res.locals.flags.articlesFromContentApiV2.isSwitchedOn ? process.env.apikey : process.env.api2key;
```
Comment (commuterjoy) [inline server/controllers/access.js]: metrics?

File: server/controllers/access.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	var api = require('ft-api-client')(apiKey);
+	if(req.method === 'HEAD' && req.get('X-FT-Access-Metadata') === 'remote_headers'){
```
Comment (commuterjoy) [inline server/controllers/access.js]: will this not just hang if these conditions aren't met. either throw an error or do a `res.send(404)` or whatever.

**Comment by aintgoin2goa** [comment]
All fair points.  Have fixed and also added some cache headers

**Comment by commuterjoy** [comment]
:+1:

**Comment by matthew-andrews** [comment]
lovely

File: server/controllers/access.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+	var apiKey = res.locals.flags.articlesFromContentApiV2.isSwitchedOn ? process.env.apikey : process.env.api2key;
+	var api = require('ft-api-client')(apiKey);
+	if(req.get('X-FT-Access-Metadata') === 'remote_headers'){
+		api.get(req.params[0]).then(function(article){
+			res.set('Cache-Control', 'public, max-age=3600, s-maxage=3600');
+			res.vary('X-FT-UID');
+			res.set('X-FT-UID', article.id);
+			res.set('X-FT-Content-Classification', article.contentClassification);
+			res.status(200).end();
+		}).catch(function(e){
```
Comment (matthew-andrews) [inline server/controllers/access.js]: this can be shorter — `.catch(next)`

**Comment by aintgoin2goa** [comment]
did I merge this by mistake, or was it someone else?

**Pull request merged**